quirks/password-rules.json

```diff
@@ -566,6 +566,9 @@
     "secure.orclinic.com": {
         "password-rules": "minlength: 6; maxlength: 15; required: lower; required: digit; allowed: ascii-printable;"
     },
+    "secure.snnow.ca": {
+        "password-rules": "minlength: 7; maxlength: 16; required: digit; allowed: lower, upper, digit;"
```

Comment:
You can omit “digit” from the allowed. All required classes are allowed.

Reply:
Thanks!